exporter/datadogexporter/examples/k8s-chart/configmap.yaml

```diff
@@ -12,6 +12,22 @@ data:
         protocols:
           grpc:
           http:
+        # # Comment out this block below to get access to system metrics regarding
+        # # the OpenTelemetry Collector and its environment, such as spans or metrics
+        # # processed, running and sent, queue sizes, uptime, k8s information
+        # # and much more.
+        #
+        # prometheus:
+        #   config:
+        #     scrape_configs:
+        #     - job_name: 'otelcol'
+        #       scrape_interval: 10s
+        #       static_configs:
+        #       - targets: ['0.0.0.0:8888']
+        #       metric_relabel_configs:
+        #       - source_labels: [ __name__ ]
+        #         regex: '.*grpc_io.*'
```

Comment:
@dineshg13 why are we ignoring this one? Id like to leave a note (or simply allow it by removing this block).